draft-ietf-privacypass-rate-limit-tokens.md

```diff
@@ -790,10 +808,18 @@ forwarding it to the Issuer.
 
 If the Attester detects this Client has changed their Client Key more frequently than allowed
 as described in {{attester-state}}, it SHOULD reject the request without forwarding it to
-the Issuer.
+the Issuer, and also use this event to penalize the Client (see {{penalization}}).
 
 ## Attester-to-Issuer Request {#request-two}
 
+The Attester and the Issuer MUST use a secure and Issuer-authenticated HTTPS
+connection for all requests. Also, Issuers MUST authenticate Attesters, either via mutual
+TLS or another form of application-layer authentication. They MAY additionally use
+mechanisms such as TLS certificate pinning, to mitigate the risk of channel
+compromise; see {{sec-considerations}} for additional about this channel.
```

Comment:
Can we file an issue against the architecture draft to move this text there? We could then replace this text with "see architecture document about requirements for this channel."

Reply:
The architecture doesn't require the attester and issuer to be back-to-back connected though?

Reply:
It doesn't, but we talk about the requirements when they are split like this, so it seems like a good place to put that text.